frontend/src/app/shared/services/notification/notification.service.ts

```diff
@@ -6,21 +6,21 @@ import { NotifierService } from 'angular-notifier';
   providedIn: 'root'
 })
 export class NotificationService {
-  constructor(private _service: NotifierService ) { }
+  constructor(private service: NotifierService ) { }
 
   success(msg: string) {
```

Comment:
i think 'msg' is not good choice for name. Usage of vowels is not forbidden for us

Reply:
lol